.gitignore

```diff
@@ -38,7 +39,8 @@ node_modules
 .LSOverride
 
 # Icon must ends with two \r.
-Icon

+Icon
```

Comment:
I think the VS code editor somehow killed the special character in this `.gitignore` file.